webrecorder/browsertrix
# fix: Update org status banner on quota reached

Fixes https://github.com/webrecorder/browsertrix/issues/1954

### Changes

- Refactors app state to include org data
- Fixes banner not showing if storage or execution minutes is exceeded after page load
- Disables closing banners
- Refreshes org when tab changes

### Manual testing

1. Log in as superadmin
2. Update an org to have low storage and execution minutes quotas, but not over
3. Run workflow in org. Verify execution minutes overage banner is shown
4. Upload file to org. Verify storage overage banner is shown
